inc/cropper/src/cropper.js

```diff
@@ -158,6 +183,38 @@ Media.events.on( 'frame:select:init', frame => {
             if ( reset ) {
               frame.reset();
             }
+
+            // Update current block if we can map the attachment to attributes.
+            if ( wp && wp.data ) {
+              const selectedBlock = wp.data.select( 'core/block-editor' ).getSelectedBlock();
+              if (
+                ! selectedBlock ||
+                ! window.SmartMedia.ImageBlockAttributeMaps ||
+                ! window.SmartMedia.ImageBlockAttributeMaps[ selectedBlock.name ]
```

Comment:
😢  can we single-line this?

Reply:
sometimes you complain about long lines! But yes, sure

Reply:
I'll do both! I'd prefer we split the `if` into logical groups even if it's duplicative to ensure readability and mental parsing. We could also alias `window.SmartMedia.ImageBlockAttributeMaps` into a shorter var.